src/Popover/Popover.d.ts

```diff
@@ -24,7 +24,7 @@ export type PopoverProps = {
   transformOrigin?: Origin;
   transitionDuration?: number | 'auto';
   theme?: Object;
-} & Partial<TransitionHandlers> &
-  PaperProps;
+  PaperProps?: Partial<PaperProps>;
```

Comment:
`Partial` is not needed, but also doesn't hurt, I guess :)

Reply:
just to be future proof in case paper ever gets better required prop :)

Reply:
just to be future proof in case paper ever gets better required prop :)

Reply:
better = a, damn phone keyboard

Reply:
Good thinking.